templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -330,8 +330,20 @@ contents:
     }
 
     stop_all_containers() {
-      echo "Stopping all containers.."
-      crictl ps -q | xargs -r crictl stop
+      conids=$(crictl ps -q)
+      iterations=0
+      while [ ! -z "$conids" ]; do
+          let iterations=$iterations+1
+          if [ $iterations -ge 60 ]; then
+              echo "Failed to stop all containers after 60 iterations. Exiting!"
+              exit 1
+          fi
+          crictl stop $conids || true
+          echo "Waiting for all containers to stop... ($iterations/60)"
+          sleep 5
+          conids=$(crictl ps -q)
+      done
+      echo "All containers are stopped."
```

Comment:
Added a message at the end, so that we provide information to the user even if we don't get into the while loop.